filters.json

```diff
@@ -38,5 +38,15 @@
         "description": "Issues that are unresolved", 
         "id": "12323020", 
         "jql": "resolution is EMPTY"
+    },
+    "ds_decisions_maint": {
+        "description": "DECISIONs for current project+versions for JBIDE/JBDS maintenance", 
+        "id": "12323026", 
+        "jql": "((project = JBIDE AND fixVersion in ( %(jbide_maint_versions)s )) OR (project = JBDS AND fixVersion in ( %(jbds_maint_versions)s ))) AND comment ~ DECISION"
```

Comment:
jira queries are not case sensitive. 

thus comment ~ "DECISION" returns results even if I write "we need a decision"...thus this query won't be very useful IMO (unless we ban the use of the word decision ;)